File: app/src/main/java/com/julia/tag/MenuActivity.java
package com.julia.tag;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;

import com.julia.tag.game.GameActivity;
import com.julia.tag.records.RecordsActivity;

public class MenuActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_menu);

        findViewById(R.id.activity_menu_play).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MenuActivity.this, GameActivity.class));
                    overridePendingTransition(R.anim.pull_enter, R.anim.pull_exit);
            }
        });

        findViewById(R.id.activity_menu_records).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MenuActivity.this, RecordsActivity.class));
                overridePendingTransition(R.anim.pull_enter, R.anim.pull_exit);
            }
        });

        findViewById(R.id.activity_menu_exit).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: Fahrenheit/app/src/main/java/com/example/fahrenheit/MenuActivity.kt
package com.example.fahrenheit

import android.content.Context
import android.content.Intent
import android.graphics.Color
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_menu.*


class MenuActivity : AppCompatActivity() {

    private var isLight = true

    override fun onResume() {
        super.onResume()
        val pref = getSharedPreferences("LIGHT_MOD", Context.MODE_PRIVATE)
        isLight = pref.getBoolean("LIGHT", false)
        isLightMode()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu)
        startTheGame()
        gameProgress()
        settings()
    }

    private fun isLightMode() {
        if (isLight) {
            constraint_menu!!.setBackgroundColor(Color.WHITE)
            start_game_btn.setTextColor(Color.BLACK)
            progress_game_btn.setTextColor(Color.BLACK)
            settings_btn.setTextColor(Color.BLACK)
        } else {
            constraint_menu!!.setBackgroundColor(Color.BLACK)
            start_game_btn.setTextColor(Color.WHITE)
            progress_game_btn.setTextColor(Color.WHITE)
            settings_btn.setTextColor(Color.WHITE)
        }
    }

    private fun gameProgress() {
        progress_game_btn.setOnClickListener {
            startActivity(Intent(this, ProgressActivity::class.java))

        }
    }

    private fun settings() {
        settings_btn.setOnClickListener {
            startActivity(Intent(this, SettingsActivity::class.java))
        }
    }

    private fun startTheGame() {
        start_game_btn.setOnClickListener {
            startActivity(Intent(this, MainActivity::class.java))
        }
    }
}
